src/data/screenshots_de.json

```diff
@@ -5,14 +5,16 @@
     },
     "section-main": {
         "headline": {
-            "title": "Machen Sie sich selbst ein Bild (Version 2.8)"
+            "title": "Machen Sie sich selbst ein Bild (Version 2.9)"
         },
         "content": {
             "textblock": [
-                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.8 (verfügbar seit 25.08.2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
-                "Neue Funktionen in Version 2.8 sind die Standardisierung der Namen und die Möglichkeit, Zertifikate aus Testergebnissen zu öffnen.",
-                "Neben dem Namen in der länderspezifischen Schreibweise wird nun in jedem Zertifikat auch der Name in standardisierter, lateinischer Schreibweise angezeigt.",
-                "Sie können nun in der App direkt aus dem Testergebnis Ihr digitales COVID-Testzertifikat der EU abrufen, sofern ein Zertifikat verfügbar ist. Diese Funktion ist bisher nur für Android verfügbar, wird aber zukünftig auch für iOS verfügbar sein.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.9 (verfügbar seit 08.09.2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Neue Funktionen in Version 2.9 sind das Warnen in Vertretung, die sofortige Impfstatusanzeige für Genesene, der Aufruf von digitalen COVID-Testzertifikaten der EU aus einem Testergebnis (zuvor nur auf Android möglich) und das Hinzufügen einer Auffrischimpfung.",
```

Comment:
In the general public the term "Auffrischungsimpfung" is used, the BZGA uses it: https://www.infektionsschutz.de/coronavirus/schutzimpfung/fragen-und-antworten/auffrischungsimpfung.html.
So maybe we should replace all occurrences of "Auffrischimpfung" with "Auffrischungsimpfung"?

Reply:
Changed in next commit, thanks